### PR TITLE
Release v2.0.5 rc1

### PR DESCRIPTION
**Pull request by Eirik0**

### CODE REVIEW

File: doc/release-notes/release-notes-2.0.5-rc1.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Larry Ruane (6):
+      fix enable-debug build DB_COINS undefined
+      add -addressindex changes for bitcore insight block explorer
+      add -spentindex changes for bitcore insight block explorer
+      Update boost from v1.69.0 to v1.70.0. #3947
+      add -timestampindex for bitcore insight block explorer
+      3873 z_setmigration cli bool enable arg conversion
+
+Marius Kjærstad (1):
+      Update _COPYRIGHT_YEAR in configure.ac to 2019
```
Comment (LarryRuane) [inline doc/release-notes/release-notes-2.0.5-rc1.md]: The underscore before COPYRIGHT causes the rest of this file to be italicized. 
```suggestion
      Update \_COPYRIGHT_YEAR in configure.ac to 2019
```

File: src/deprecation.h [hunk 2]
```diff
@@ -8,7 +8,7 @@
 // Deprecation policy:
 // * Shut down 16 weeks' worth of blocks after the estimated release block height.
 // * A warning is shown during the 2 weeks' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 504600;
+static const int APPROX_RELEASE_HEIGHT = 525525;
```
Comment (LarryRuane) [inline src/deprecation.h]: I think this is approximately May 2, 7 am MDT

File: doc/release-notes/release-notes-2.0.5-rc1.md (hunk 1 above)
Comment (Eirik0) [reply doc/release-notes/release-notes-2.0.5-rc1.md]: I do see what you are seeing in the diff, but I do not see that when viewing the actual page: https://github.com/zcash/zcash/blob/12a9e172e5901c2ffa428fa52790eb1453e0b0c0/doc/release-notes/release-notes-2.0.5-rc1.md. We could definitely do something to make this render better, but we have similar rendering issues in other release notes' markdown files. I am OK with leaving this as is.

**Comment by bitcartel** [comment]
The experimental developer option to set shielded pool size to zero, for the purpose of testing the tunstile, should be documented in the release notes.  Along with info about turnstile activation on mainnet, which is a notable change.

When running the release process script, the documentation here https://github.com/zcash/zcash/blob/master/doc/release-notes.md is moved into the folder https://github.com/zcash/zcash/tree/master/doc/release-notes with a changelog appended.

**Comment by bitcartel** [comment]
The above doesn't block, as `release-notes.md` can be updated in a PR before the final 2.0.5 release.

**Comment by Eirik0** [comment]
@zkbot r+

**Comment by mdr0id** [comment]
@zkbot r+

**Pull request merged**